packages/react-scripts/template/README.md

```diff
@@ -1452,6 +1452,7 @@ If this doesn’t happen, try one of the following workarounds:
 * Some editors like Vim and IntelliJ have a “safe write” feature that currently breaks the watcher. You will need to disable it. Follow the instructions in [“Working with editors supporting safe write”](https://webpack.github.io/docs/webpack-dev-server.html#working-with-editors-ides-supporting-safe-write).
 * If your project path contains parentheses, try moving the project to a path without them. This is caused by a [Webpack watcher bug](https://github.com/webpack/watchpack/issues/42).
 * On Linux and macOS, you might need to [tweak system settings](https://webpack.github.io/docs/troubleshooting.html#not-enough-watchers) to allow more watchers.
+* On Windows run `npm i --save-dev cross-env` and then `cross-env CHOKIDAR_USEPOLLING=true npm start` ([discussed here](https://github.com/facebookincubator/create-react-app/pull/1608#discussion_r102511341)).
```

Comment:
This makes it sound like you have to *always* do that to resolve the issue rather than only when you use VirtualBox. What I meant is to *replace* the previous bullet point you added with this instruction, not to add another bullet. There's also no need to mention Windows specifically because with `cross-env` the same command will just work everywhere.